tex/latex/pgf/utilities/pgfpages.sty

```diff
@@ -34,6 +34,7 @@
 \pgfphysicalpageemptytrue
 \pgf@holdingphysicalpagefalse
 
+\let\pgfpagespoint\pgfpoint
```

Comment:
With some auto detections:
```suggestion
\def\pgfpagespoint{%
  % Delay the definition of \pgfpagespoint in case the \pagedir is set at the
  % beginning of document, but still assume that the \pagedir is constant 
  % throughout the document.
  \pgfpages@test@pagedir\pgfpagespoint}

\def\pgfpages@test@pagedir{%
  \let\pgfpagespoint\pgfpoint
  \ifdefined\luatexversion
    % \pagedir returns a three-char string and \pagedirection returns an int
    \ifcase\pagedirection
        % 0, TLT (latin)
    \or % 1, TRT (arabic)
      \def\pgfpagespoint##1##2{\pgfpoint{##1}{##2}\pgf@x=-\pgf@x}%
    \or % 2, LTL (mongolian)
      % ... need any transformations?
    \or % 3, RTT (cjk)
      % ... need \pgf@x=-\pgf@x ?
    \fi
  \fi
}
```